packages/cli/commands/upload.js

```diff
@@ -58,6 +58,55 @@ const logThemePreview = (filePath, accountId) => {
 };
 
 exports.handler = async options => {
+  async function uploadFiles(
```

Comment:
nit: since this inner function is being passed options and not using the variable from the outer function's scope it might be cleaner to separate it out and make `exports.handler` a bit shorter.